remoteworkitem/trackerquery_repository_test.go

```diff
@@ -81,20 +91,26 @@ func (test *TestTrackerQueryRepository) TestExistsTrackerQuery() {
 		err := test.trackerRepo.Create(ctx, &tracker)
 		require.NoError(t, err)
 
-		query, err := test.queryRepo.Create(ctx, "abc", "xyz", tracker.ID, space.SystemSpace)
-		require.NoError(t, err)
+		query := remoteworkitem.TrackerQuery{
+			Query:     "abc",
+			Schedule:  "xyz",
+			TrackerID: tracker.ID,
+			SpaceID:   testFxt.Spaces[0].ID,
+		}
+		err = test.queryRepo.Create(ctx, &query)
+		assert.Nil(t, err)
 
 		err = test.queryRepo.CheckExists(ctx, query.ID)
-		require.NoError(t, err)
+		assert.Nil(t, err)
```

Comment:
Why do you replace the correct one with `Nil` again? In 13a70a7845b51b0792702e60bd96c7afad797b94 I  replaced all these with `.NoError` or `.Error`. You can look at my commit to see the command that I've used to make the change.

Reply:
corrected here 4cee394